src/components/AddBook.js

```diff
@@ -1,14 +1,41 @@
-import React from 'react';
-
-const AddBook = () => (
-  <div className="form-div">
-    <form>
-      <h3 className="new-book-title">Add New Book</h3>
-      <input type="text" placeholder="Book Title" />
-      <input type="text" placeholder="Author Name" />
-      <button type="button" className="book-btn">Add Book</button>
-    </form>
-  </div>
-);
-
-export default AddBook;
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { AddBook } from '../redux/books/books';
+
+const Addbook = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const dispatch = useDispatch();
+
+  const AddBookHandler = (e) => {
+    e.preventDefault();
+    const book = {
+      id: uuidv4(),
+      title,
+      author,
+    };
+    dispatch(AddBook(book));
+    setTitle('');
+    setAuthor('');
+  };
+
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/203621607-6a57b340-dc7a-41f6-804d-bdf1aeb74a4c.PNG)

**[Optional]**

- [ ] When you add a book then an error comes in your console window. Kindly fix this :blush: